elements/pfe-band/src/pfe-band--lightdom.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../pfe-sass/pfe-sass";
+
+// For each section in the band layout
+@each $section in (aside, header, body, footer) {
+     [slot="pfe-band--#{$section}"] { // i.e., pfe-band--header
+      // Other browsers are getting grid-area through shadow-dom
+      // Only need to target edge in the lightdom
+      @supports (display: -ms-grid) {
+          // Apply the section as a named grid-area
+          grid-area: #{$section};
+      }
+    }
+}
+
+
+@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+    pfe-band > * > * {  // target children of body area
+      width: 51%; // over 50% so if multiple items in body they will stack
+      margin-right: 4.5%;
+      float: left;
+    }
+    pfe-band [slot="pfe-band--header"],[slot="pfe-band--footer"] { // reset width of children in header & footer
```

Comment:
I think this selector needs to be `pfe-band [slot="pfe-band--header"], pfe-band [slot="pfe-band--footer"]`